### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '<'

### DISCUSSION THREAD

**Discussion by inspectorsherlock** [jeffreylanters/react-unity-webgl#533] [Help Wanted!] [answered]
![image](https://github.com/jeffreylanters/react-unity-webgl/assets/11412893/65afd29f-b277-4b30-97ae-8f8102b3081f)
![image](https://github.com/jeffreylanters/react-unity-webgl/assets/11412893/8a4d2566-12f2-4450-897c-99846cb618fb)

If anyone can view this, what is the issue with this UnityWebGL console error? I don't understand what it wants me to do? 
The problem says there is an unexpected token, what is that telling me? Is it that the <!DOCTYPE html> has a wrong "<" placed somewhere within the 4 data types provided after building through WebGL ? The reason being is that it is completely preventing myself from even seeing it load: it just gives me a blank web page. 

The console states there's something wrong but I can't do anything about it. To be clear, I have placed my Unity build within the public folder and the console still throws the same error. I used disabled compression for this, for some reason it still gives this weird token error.

If anyone understands what the solution is for this error, please let me know. I have react working and Unity working however it seems like there is something up with the "Bridging system" between the two. It's a shame because it's got the potential for a good combination.

**Comment by jeffreylanters** [comment, marked as answer]
Hi. Could you please try removing "public" from your URLs.

**Reply by inspectorsherlock** [reply]
Thank you for your swift response, it works now. Thank you so much, much appreciated. 👍🏼😅
